standard/links/commons/links.lua

```diff
@@ -349,6 +349,112 @@ local ICON_KEYS_TO_RENAME = {
 	douyin_page = 'douyin',
 }
 
+local MATCH_ICONS = {
+	breakingpoint = {
+		icon = 'File:Breaking Point GG icon lightmode.png',
+		iconDark = 'File:Breaking Point GG icon darkmode.png',
+		text = 'Breaking Point matchpage'
+	},
+	cdl = {
+		icon = 'File:Call of Duty League lightmode.png',
+		iconDark = 'File:Call of Duty League darkmode.png',
+		text = 'Call of Duty League matchpage'
+	},
+	challengermode = {icon = 'File:Challengermode icon.png', text = 'Match page on Challengermode'},
+	datdota = {
+		icon = 'File:DatDota-icon.png',
+		text = 'datDota'
+	},
```

Comment:
is there a reason that somne are on one line while others have line breaks?
(not talking about the longer ones which naturally need the line breaks)

Reply:
Copied from their respective current data, so whatever it was before. Probably will change it

Reply:
Changed